docs/reference/data-tables.md

```diff
@@ -182,3 +182,145 @@ numbers, filesizes, dates and month names. See the [tablesort documentation]
 
   [tablesort]: http://tristen.ca/tablesort/demo/
   [instant loading]: ../setup/setting-up-navigation.md#instant-loading
+
+### Import table from file
+
+[:octicons-cpu-24: Plugin][table-reader-docs]
+
+You can also import data from a CSV or Excel file using the plugin [`mkdocs-table-reader-plugin`][table-reader-docs].
+
+First, you will need to install it with `pip`:
+
+```sh
+pip install mkdocs-table-reader-plugin
+```
+
+Then extend the `mkdocs.yml` file like this:
+
+```yaml
+plugins:
+  - table-reader
+```
+
+Then, you can import the data like this:
+
+=== "CSV"
+    
+    For this example, let's use a CSV in the local directory.
+
+    === "Data"
+        
+        The data file may look like this:
+
+        ```csv title="./data.csv"
+        col1,col2,col3
+        r1c1,r1c2,r1c3
+        r2c1,r2c2,r2c3
+        r3c1,r3c2,r3c3
+        ```
+
+    === "Markdown"
+
+        And you can add it to your Markdown page like this:
+
+        ```md title="./markdown.md"
+        ...
+        {{ read_csv('./data.csv') }}
+        ...
+        ```
+
+    === "Result"
+
+        And it will then return a result like this:
+
+        col1|col2|col3
+        ----|----|----
+        r1c1|r1c2|r1c3
+        r2c1|r2c2|r2c3
+        r3c1|r3c2|r3c3
+
+=== "Excel"
```

Comment:
Please don't use double nested tabs. That's an anti-pattern. Yes, we also use it in 1-2 sections, but I think this section could be easily split in two sections with headlines + one tabbed container.